src/util/util.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+
+def mkdir_if_not_exist(inputdir):
+    if not os.path.exists(inputdir):
+        os.makedirs(inputdir)
+    return inputdir
+
+def run_unix_cmd(cmd, verbose=True):
```

Comment:
already one or should be merged with parrun() which I use to run solo comamnds as well